### DISCUSSION TITLE
How can I have access to CrateDB from another server?

### DISCUSSION THREAD

**Discussion by computer-8256** [crate/crate#16585] [General]
I installed CrateDB on a Debian OS using deb repository and apt. I want to have access to it from another server on same network provider.  when I open the ip in a browser, nginx come up. but, when I add the port to the address, I get `ERR_CONNECTION_REFUSED`.

**Comment by computer-8256** [comment]
I need both web interface and PSQL. ports to work.

**Comment by seut** [comment]
By default, CrateDB's HTTP port is set to 4200 and PSQL is set to 5432. Both ports must be accessible from the other machine to use both interfaces. See https://cratedb.com/docs/crate/reference/en/latest/config/node.html#ports and https://cratedb.com/docs/crate/reference/en/latest/interfaces/index.html.
